src/temp/ProjectInfo/ExternalCertification.tsx

```diff
@@ -0,0 +1,62 @@
+import { ProjectExpense } from '@planet-sdk/common';
+import styles from './ProjectInfo.module.scss';
+import { useTranslation } from 'next-i18next';
+import DownloadIcon from '../icons/DownloadIcon';
+import SingleProjectInfoItem from './SingleProjectInfoItem';
+import ExternalCertificationItem from './ExternalCertificationItem';
+import ProjectSpendingCertificationItem from './ProjectSpendingCertificationItem';
+import ProgressReportCertificationItem from './ProgressReportCertificationItem';
+
+export const renderDownloadIcon = () => {
+  return (
+    <div className={styles.downloadIcon}>
+      <DownloadIcon
+        width={10}
+        color={`${'rgba(var(--certification-background-color-new))'}`}
+      />
+    </div>
+  );
+};
+
+interface ExternalCertificationProps {
+  certification: string;
+  spendings: ProjectExpense[];
+  progressReports: number[];
+}
+
+const ExternalCertification = ({
```

Comment:
Since this does not include only External Certifications, let's call this the `ProjectDownloads` component.